src/common/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,78 @@
+import cn from 'classnames';
+import React from 'react';
+
+import Flex from 'common/components/Flex/Flex';
+
+import styles from './slider.scss';
+
+interface IProps {
+	slides: any[];
+	currentSlide: number;
+	onSlide?: any;
+}
+
+export const Slider: React.FC<IProps> = ({slides, currentSlide, onSlide}) => {
+	const slidePosition = (index: number) => {
+		const diff = (index + 1) - (currentSlide + 1);
+		if (index !== currentSlide) {
+			return {
+				transform:
+					`translateX(
```

Comment:
make an scss class and mote it there. User classnames module to manage styles

Reply:
15 lines of JS or 115 of SCSS? (Choose correct answer below)